lib/takeaway.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'menu'
+require_relative 'order'
+require_relative 'sms'
+class Takeaway
+
+  attr_reader :menu, :order
+
+  def initialize(menu_klass = Menu.new, order_klass = Order.new, sms_klass = Sms.new)
+    @menu = menu_klass
+    @order = order_klass
+    @sms = sms_klass
+    @total_dishes = 0
+
+  end
+
+  def select(item, qty=1)
+    fail "Sorry we don't have #{item} on the menu" unless @menu.dishes.include? item
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)